client/src/components/ThemeText/ThemeText.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./ThemeText.css";
+
+export const HeaderText = ({ data }) => {
+  return <div className="headerText"> {data} </div>;
+};
+
+export const SubtitleText = ({ data }) => {
+  return <div className="subtitleText"> {data} </div>;
```

Comment:
Using the word `text` would probably be more descriptive than just `data`

Reply:
Fixed (Nestor): changed `data` to `text`.